libs/sdk-core/src/persist/transactions.rs

```diff
@@ -178,7 +178,7 @@ impl SqliteStorage {
         let con = self.get_connection()?;
         let mut prep_statement = con.prepare(
             "
-        INSERT INTO sync.open_channel_payment_info (
+        INSERT OR IGNORE INTO sync.open_channel_payment_info (
```

Comment:
No problem with that, just curious if you encountered this error in a real run.

Reply:
I see... now that the call was moved to wrap_open_channel_invoice.

Reply:
Yeah it can be called multiple times, so should be idempotent